docs/install/config-ha.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Configuring high-availability components"
+linkTitle: "Configuring high-availability components"
+weight: 50
+type: "docs"
+---
+
+Active/passive high availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs. In an HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
+
+Active/passive HA in Knative is available through leader election, which is enabled by default after the Knative Serving control plane is installed.
+
+When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required. These controller instances compete to use a shared resource, known as the leader election lock. The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
+
+HA functionality is available on Knative for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, `kourier-gateway`, `istio-pilot`, and `istio-gateway` components.
+
+## Enabling leader election
+
+1. Enable leader election for the control plane controllers:
+```
+kubectl patch configmap/config-leader-election \
+  --namespace knative-serving \
+  --type merge \
+  --patch '{"data":{"enabledComponents": "controller,hpaautoscaler,certcontroller,istiocontroller,nscontroller"}}'
```

Comment:
This won't change Kourier, you would need to run a different command for that that touched the same configmap, but in the `kourier-system` namespace.

Reply:
Should we add this then? Could you provide the command and any other details please?

Reply:
Just gonna remove all mentions of Kourier etc for now, we can do some follow up stuff to flesh out any docs we need for that later.